### DISCUSSION TITLE
Error when re-using .quant files from older DIA-NN version?

### DISCUSSION THREAD

**Discussion by MalteSielaff** [vdemichev/DiaNN#494] [Q&A] [answered]
Hi Vadim,

I'm trying to perform a combined DIA-NN analysis of and old and a more recent experiment. Both experiments were already processed using similar parameters and the same predicted spectral library, the older data using DIA-NN v1.8 and the recent data using DIA-NN v1.8.1. My idea was to re-use the .quant files in a combined analysis but unfortunately there seems to be a problem:

[...]
[0:25] Cross-run analysis
[0:25] Reading quantification information: 1 files
ERROR: the .quant file for the run 74.dñ?qk·¾±$í?5PÂCj¸ç?$=6×ê?	S¾"Ñ>¾DVDòNÙB‡ô>mE?O@žDJ€DpKO?€–ËBÁh?Eºk?€–Ë?ÿ>ŸÓ?EP 
EØ(ËCÔì>C?C>BC¢>L=A–Ð>	?B?A?	d?
ØY?‹5Z?•^d?H›>D?5´ß??-¿	B­Îï¾HpišGþ?€–Ë?(v?•=~?­!ûCA.Fez?G]^
[...]

I'm not sure, but I think that the .quant files generated by the different versions of DIA-NN are not compatible. Can you confirm that? If not, do you have any idea what could be causing the problem?

Best regards,
Malte

[DIANN_1.8.1_log_ErrorQuantFile_09_05_2022 16_08_58.txt](https://github.com/vdemichev/DiaNN/files/9506393/DIANN_1.8.1_log_ErrorQuantFile_09_05_2022.16_08_58.txt)

**Comment by vdemichev** [comment, marked as answer]
Hi Malte,

Unfortunately, .quant files are indeed not compatible between different DIA-NN versions.

Best,
Vadim

**Reply by MalteSielaff** [reply]
Thank you Vadim! Reprocessing of the old data is already in progress :)

Best regards,
Malte
